repos/aspnet-extensions.proj

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <PreferredHash>48cb2abb441bfa3affbcd9ae0988af05b2416e03</PreferredHash>
+    <SourceDirectory>Extensions</SourceDirectory>
+  </PropertyGroup>
+
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))/dir.props" />
+
+  <PropertyGroup>
+    <OfficialBuildId>20190529.9</OfficialBuildId>
+
+    <BuildCommandArgs/>
+    <BuildCommandArgs>$(BuildCommandArgs) --configuration $(Configuration)</BuildCommandArgs>
+    <BuildCommandArgs>$(BuildCommandArgs) --binaryLog</BuildCommandArgs>
+    <BuildCommand>$(ProjectDirectory)build$(ShellExtension) $(BuildCommandArgs)</BuildCommand>
+
+    <PackagesOutput>$(ProjectDirectory)artifacts/packages/$(Configuration)/Shipping/</PackagesOutput>
```

Comment:
There are going to be some packages in `artifacts/packages/$(Configuration)/NonShipping/` as well which are used to transport bits upstream to other repos. Is it possible to define multiple package output paths?

Reply:
Not currently, but I think we should be able to workaround this using the output placement API - I'm giving this a try now.